spot-contracts/contracts/BondIssuer.sol

```diff
@@ -113,7 +124,35 @@ contract BondIssuer is IBondIssuer, OwnableUpgradeable {
 
     /// @inheritdoc IBondIssuer
     function isInstance(IBondController bond) external view override returns (bool) {
-        return _issuedBonds.contains(address(bond));
+        return (_activeBonds.contains(address(bond)) || _matureBonds.contains(address(bond)));
+    }
+
+    /// @inheritdoc IBondIssuer
+    /// @dev Reverts if none of the active bonds are mature.
+    function matureAll() external {
+        bool bondsMature = false;
+        uint256 activeCount_ = _activeBonds.length();
```

Comment:
Do we use the trailing suffix consistently? Is it supposed to mean local scope?

Reply:
We usually use the trailing `_` when there is a namespace conflict. There's already a method called `activeCount` and hence the variable is `activeCount_`.

The `_` when prepended to a variable indicates its private.